docker/alpine/Dockerfile

```diff
@@ -42,6 +42,7 @@ USER builder
 RUN cd /dist \
 	&& abuild-keygen -a -n \
 	&& abuild checksum \
+	&& git init \
```

Comment:
What's this for?

Reply:
This is a work around for the current Alpine package builder `abuild`. It expects to run in a git repository otherwise fails.
I know it's not pretty and am very open for alternatives.